package.json

```diff
@@ -17,7 +17,7 @@
     "url": "https://github.com/microsoft/vscode-jupyter/issues"
   },
   "engines": {
-    "vscode": "^1.55.0"
+    "vscode": "^1.56.0"
```

Comment:
I believe the proposed API changes below haven't gone to stable yet. Does this release need to be for "^1.57.0-insider" instead?

Reply:
that sucks, I guess I'll make this an insiders only release

Reply:
Yeah I think Claudia is only hitting this in Insiders (I have Gather installed in Stable and have no issues opening notebooks), so I would guess putting this change into Stable would break people there.

Reply:
We will probably also need to publish a "^1.57.0" release too.

Reply:
@DonJayamanne so -insiders release currently show up in stable, but stable releases are also available to insiders, right? Could we just make this a 1.57.0 release and drop the "-insiders"? There isn't a 1.57 stable release, and when there is one is should use the changes here. I think that's what I would do, but wanted to check before recommending that.

Reply:
If we made this a 1.57.0 release, I think customers on 1.57.0-insider wouldn't be able to install this VSIX. Is that what we want?